spec/rubocop/cop/lint/boolean_symbol_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::Lint::BooleanSymbol, :config do
+  subject(:cop) { described_class.new(config) }
+
+  it 'registers an offense when using `:true`' do
+    expect_offense(<<-RUBY.strip_indent)
+      :true
+      ^^^^^ Symbol with a boolean name - you probably meant to use `true`.
+    RUBY
+  end
+
+  it 'registers an offense when using `:false`' do
+    expect_offense(<<-RUBY.strip_indent)
+      :false
+      ^^^^^^ Symbol with a boolean name - you probably meant to use `false`.
+    RUBY
+  end
+
+  it 'do not registers an offense when using regular symbol' do
```

Comment:
Would it provide value to add a test for the boolean versions of true and false?